test/probes/express.test.js

```diff
@@ -88,41 +195,105 @@ describe('probes.express ' + pkg.version, function () {
       function (msg) {
         check['express-entry'](msg)
       },
+      // skip the reqRoutePath profile entry and exit
       function () {},
-      function () {},
+      function () {}
+    ]
+
+    // if it is get then skip the '*' profile entry and exit too.
+    // (it will be called first, but we need two more skips anyway.)
+    if (method === 'get' || true) {
+      validations = validations.concat([function () {}, function () {}])
+    }
+
+    validations = validations.concat([
       function (msg) {
         check['express-exit'](msg)
       },
       function (msg) {
         check['http-exit'](msg)
-        msg.should.have.property('TransactionName', 'express.GET/hello/:name')
-        msg.should.have.property('Controller', '/hello/:name')
-        msg.should.have.property('Action', 'hello')
+        msg.should.have.property('TransactionName', expected('tx'))
+        msg.should.have.property('Controller', expected('c'))
+        msg.should.have.property('Action', expected('a'))
       }
-    ]
+    ])
+
     helper.doChecks(emitter, validations, function () {
       server.close(done)
     })
 
     var server = app.listen(function () {
       var port = server.address().port
-      request('http://localhost:' + port + '/hello/world')
+      var options = {
+        url: 'http://localhost:' + port + (method === 'get' ? '/hello/world' : '/api/set-name')
+      }
+      if (method === 'post') {
+        options.json = {name: 'bruce'}
+      }
+      request[method](options)
     })
-  })
+  }
 
+  //
+  // custom transaction names
+  //
   it('should allow a custom TransactionName', function (done) {
-    var app = express()
+    // supply a simple custom function
+    function custom (req, res) {
+      var result = 'new-name.' + req.method + req.route.path
+      return result
+    }
 
-    conf.makeMetricsName = function (req, res) {
-      return {
-        Controller: 'new-name',
-        Action: req.method + req.route.path
-      }
+    customTransactionName(custom, done)
+  })
+
+  it('should allow a custom TransactionName with domain prefix', function (done) {
+    // simple custom function
+    function custom (req, res) {
+      var result = 'new-name.' + req.method + req.route.path
+      return result
     }
 
-    app.get('/hello/:name', function hello(req, res) {
+    ao.cfg.domainPrefix = true
+    try {
+      customTransactionName(custom, done)
+    } finally {
+      ao.cfg.domainPrefix = false
+    }
+  })
+
+  it('should handle an error in the custom name function', function (done) {
+    function custom (req, res) {
+      throw new Error('I am a bad function')
+    }
+    customTransactionName(undefined, done)
```

Comment:
should this be
 `customTransactionName(**custom**, done)`

Reply:
Yes, it should be. Thanks for catching this.